src/Common/InlineSvg.ts

```diff
@@ -0,0 +1,17 @@
+class InlineSvg {
+  private readonly inlineSvg: string;
+  private readonly viewBoxWidth: number;
+  private readonly viewBoxHeight: number;
+
+  constructor(inlineSvg: string, viewBoxWidth: number, viewBoxHeight: number) {
+    this.inlineSvg = inlineSvg;
+    this.viewBoxWidth = viewBoxWidth;
+    this.viewBoxHeight = viewBoxHeight;
+  }
+
+  public render(width: number): string {
+    return `<svg width="${width}" viewBox="0 0 ${this.viewBoxWidth} ${this.viewBoxHeight}">${this.inlineSvg}</svg>`;
```

Comment:
The render section here might be better suited for puzztool

Reply:
I contemplated that, but fundamentally anyone who consumes this data would have to write such a function.  If we ever hit a point where we'd need to consume the raw data separately, I'd be up for expanding this then, but for the moment exposing the raw components when there's only one obvious way to put them together seemed weird.